hashing/ripemd_160.cpp

```diff
@@ -0,0 +1,457 @@
+/**
+ * @file
+ * @author [Shree Harish S](https://github.com/ShreeHarish)
```

Comment:
author should preferably be the last tag in the header file

Reply:
I saw author at the top for other files in the hashing directory. Took reference from that. I will change it to the bottom of the header